5-infrastructure/business_unit_1/shared/variables.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "app_cicd_build_sa" {
+  description = "Service account email of the account to impersonate to run Terraform"
+  type        = string
+}
+
+variable "app_cicd_repos" {
+  description = "A list of Cloud Source Repos to be created to hold app infra Terraform configs"
+  type        = list(string)
+  default     = ["root-config-repo", "accounts", "transactions", "frontend"]
```

Comment:
How did we arrive at these defaults? If we are doing 1 per service, BoA has 6 services
https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/master/src

Reply:
Dima confirmed these, it is one for each namespace I believe we can bring it up Monday